include/curl/curl.h

```diff
@@ -795,6 +795,9 @@ typedef enum {
    servers, a user can this way allow the vulnerability back. */
 #define CURLSSLOPT_ALLOW_BEAST (1<<0)
 
+/* - TLSv1.3 MIDDLEBOX mode */
+#define CURLSSLOPT_TLS13_MIDDLEBOX (1<<0)
```

Comment:
Surely this shouldn't use the same bit as "allow beast" ?

Reply:
New bit used for disable tls13 middlebox.